src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityCrate.java

```diff
@@ -31,11 +36,16 @@
 import javax.annotation.Nullable;
 import java.util.List;
 
+import static gregtech.api.capability.GregtechDataCodes.IS_TAPED;
+import static gregtech.api.capability.GregtechDataCodes.TAG_KEY_PAINTING_COLOR;
+
 public class MetaTileEntityCrate extends MetaTileEntity {
 
     private final Material material;
     private final int inventorySize;
     private ItemStackHandler inventory;
```

Comment:
Can this field be made `protected`? It's useful for addons (TJCore) that are trying to extend this class